### DISCUSSION TITLE
Add option to ignore kernel vulnerabilities

### DISCUSSION THREAD

**Discussion by RobinFrcd** [aquasecurity/trivy#4816] [Ideas]
### Description

Hi,
As discussed in https://github.com/aquasecurity/trivy/discussions/3856, it would make sense to add an option to ignore kernel vulnerabilities detected by trivy. As the docker image itself doesn't ship with a kernel.

Thanks,

### Target

Container Image

### Scanner

Vulnerability

**Comment by RobinFrcd** [comment]
2 years later, this is still a relevant issue. Any plan to address that ?

**Comment by knqyf263** [comment]
Tracking https://github.com/aquasecurity/trivy/issues/9292
